tests/Makefile

```diff
@@ -269,8 +269,8 @@ endif
 
```

Comment:
With CMake all tests are added to a flat list and then executed by ctest (not with `run-test-suites.pl`) so there is no early-exit.

Reply:
Nope, cmake is not affected because it goes about it in a sane way: first, collect the list of test suites to run, then run them all (I just tested to make extra sure).